### ISSUE TITLE
❌ 🔗 🐞 Bug report: WooCommerce widgets miss translations / editable fields / styling (Duplicate of #17975)

### ISSUE THREAD

**Issue by mbaierl** [elementor/elementor#18172]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The WooCommerce cart & checkout widgets have numerous issues which prevent real world usage:

- "Coupon code" cannot be edited or translated
- "If you have a coupon code, please apply it below" cannot be edited or translated
- Buttons ignore the default button styles from Elementor and thus have to be styled individually 
- Button styles are shown in edit mode but revert to the default on the site itself

### Steps to reproduce

Simply try to create a nice looking WooCommerce checkout process in German. It's not possible. 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

n/a

**Comment by Toni-X** [comment]
Please make these translatable:
![image](https://user-images.githubusercontent.com/83016490/160118270-4ee54f2d-1a9d-4ea1-a1cf-b5095decdbb2.png)

**Comment by creativeart** [comment]
+1 . It's been a long time since I reported this bug to the Elementor team even during the beta test and the bug is still there after so much time, it's very annoying for multilingual sites

**Comment by nicholaszein** [comment]
Hello there @mbaierl!

Thank you for submitting your issue! 🙏

### ❌ While reviewing your report, we noticed it was missing some key pieces of information.

- **Clear and concise title:** You did not write a clear and concise title;
- **Prerequisites:** please search the repository before submitting an issue;
- **Steps to replicate:** You did not provide any way for us to try to reproduce your issue;
- **System info:** You did not provide any information about your [System Info](https://elementor.com/help/systems-info/) as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), we need this to be able to recreate your issue on a staging environment that would match your own.
- **One issue per thread:** We can't track multiple issues per thread. Open a new issue for each problem you're reporting here. For example, the styling bug you reported is a duplicate of #17975, which is on the same subject.

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

We are changing the way we track issues, and trying to solve old ones, while also giving the feedback you deserve. When there are multiple reports on the same issue, it becomes impossible for us to track them.

Thank you so much for your comprehension and patience! 🙏

Kind regards,

Nick
